### ISSUE TITLE
import { Form, Validator } from 'smalldots'; do not work

### ISSUE THREAD

**Issue by sibelius** [smalldots/smalldots#19]
`import { Form, Validator } from 'smalldots';`

I think it is common to be able to import component from the same global name instead of having to use different paths for different components

this works, but it is not so simple as the version above

``` js
import Form from 'smalldots/lib/Form';
import Validator from 'smalldots/lib/Validator';
```

I guess that webpack or webpack 2 are smart enough to remove files that are not used

**Comment by hnordt** [comment]
I'll keep it open for now, I'm not sure if it's necessary. I'l wait for more feedback.

**Comment by cjke** [comment]
In terms of more feedback, I would like to echo @sibelius 

Without digging, I tried the following:

    import { Fetch } from 'smalldots'; // then
    import Fetch from 'smalldots/Fetch'; // then
    import smalldots from 'smalldots'; 
    
All of which didn't work. So I came to the docs, then to issues. It's not a biggie, but a nice to have

**Comment by hnordt** [comment]
Thank you for the feedback @cjke.

This feature will be available in the next version (`v0.33`). It'll come with a whole new architecture for React apps.

It's still being tested in production, I plan to release in the next weeks.

**Comment by cjke** [comment]
Cool sounds good, keep up the top work - it's great stuff
